docs/user-guide/configuration.md

```diff
@@ -42,17 +53,6 @@ $ tree
 $ terraform-docs -c .tfdocs-config.yml .
 ```
 
-As of `v0.13.0`, the order for looking for config file is:
```

Comment:
I think it might be better to restore this back, because the logical importance of this section is lower than the previous paragraph on how to use alternative name for config file.

Reply:
ok I can work on addressing your latest feedback today

Reply:
Thanks for clarifying the implied emphasis.
How do these latest edits read to you?